components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementServiceImpl.java

```diff
@@ -366,6 +366,19 @@ public void removeClaimMappingAttributes(int tenantId, String userstoreDomain) t
         }
     }
 
+    /**
+     * Remove all claims of a given tenant.
+     *
+     * @param tenantId Id of the tenant
+     * @throws ClaimMetadataException
+     */
+    @Override
+    public void removeAllClaims(int tenantId) throws ClaimMetadataException {
+
+        // The relevant external claim deletions are handled by the DB through ON DELETE CASCADE
```

Comment:
Add proper formatting. The comment should end with a "."

Reply:
Fixed in #3312